payment-service-monitor/src/request-sender/request-sender.service.ts

```diff
@@ -55,35 +56,39 @@ export class RequestSenderService {
 
   /**
    * Makes a request based on the request parameters inside post body
-   * and evaluates whether the expected type and the received type of
+   * and evaluates whether the expected response and the received response of
    * the response match. If not, a log is created and sent to the issue creator.
    * An appropriate response with the log is additionally returned for the UI to display.
+   *
    * @param requestParams post body with request parameters
+   * @returns an object that contains a message field named msg and a log field named log.
+   * Depending on the outcome of the request, the log field is populated and the message is set
+   * accordingly
```

Comment:
für jene leser, die den code nicht kennen, verursacht die beschreibung des rückgabewertes m.e. mehr verwirrung, als dass sie hilft. sie wirft die fragen auf, in wie fern der rückgabewert vom ergebnis der anfrage abhängt und mit auf was genau 'log' gesetzt wird. wäre gut, wenn ihr da etwas mehr klarheit schafft :)

(es würde m.e. schon ausreichen, wenn ihr darauf hinweißt, dass 'log' mit dem zuvor erzeugten log befüllt wird. unter welchen umständen genau ein neues log erzeugt wird, kann der geneigte leser dann auch im fließtext nachschlagen. )

Reply:
habs geändert.